### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lesiac**

### CODE REVIEW

**Comment by Lesiac** [comment]
Hi @swati-nair I made updates to your readme file...pls review...thx!

File: README.md [hunk 1]
```diff
@@ -94,9 +94,9 @@ volumes:
...
+As a developer, if you want to build and test this operator to try out and learn more about the operator and its capabilities, you can use the following developer guide. This guide provides commands for a quick installation and initial validation for running the operator.
 
-> **Important:** The following developer guide is provided as-is and only for trial and education purposes. IBM and IBM Support does not provide any support for the usage of the operator with this developer guide. For the official supported install and usage guide for the operator, see the the IBM Knowledge Center documentation for your IBM Cloud Pak or for IBM Cloud Platform Common Services.
+> **Important:** The following developer guide is provided as-is and only for trial and education purposes. IBM and IBM Support does not provide any support for the usage of the operator with this developer guide. For the official supported instalationl and usage guide for the operator, see the the IBM Knowledge Center documentation for your IBM Cloud Pak or for IBM Cloud Platform Common Services.
```
Comment (swati-nair) [inline README.md:99]: I think there's a typo here (an extra l at the end in installation) @Lesiac

**Comment by Lesiac** [comment]
@swati-nair Thanks for catching that! All fixed!

**Comment by swati-nair** [comment]
Closing this PR as 3.6.0 release is already done. We will merge these changes once we are ready with 3.6.1 updates.

**Comment by tthorpe2** [comment]
@Lesiac @swati-nair Once we're ready to resubmit this for our csv v3.6.1, I believe the ocp version should be stated as ocp 4.3 or higher. CS 3.4.0 is not supporting ocp 4.2.

Red Hat OpenShift Container Platform **4.3** or newer installed on one of the following platforms:

**Pull request closed without merging**